src/actions/public/L2/feeActions.spec.ts

```diff
@@ -0,0 +1,182 @@
+/**
+ * The first 2 test cases are good documentation of how to use this library
+ */
+import { l2StandardBridgeABI, l2StandardBridgeAddress, optimistABI, optimistAddress } from '@eth-optimism/contracts-ts'
+import { createPublicClient, http, parseEther, parseGwei } from 'viem'
+import { optimism } from 'viem/chains'
+import { formatEther } from 'viem/utils'
+import { beforeEach, expect, test, vi } from 'vitest'
+import { estimateFees, estimateL1Fee, estimateL1GasUsed } from './feeActions'
+
+vi.mock('viem', async () => {
+  const _viem = (await vi.importActual('viem')) as any
+  return {
+    ..._viem,
+    // no way to get historical gas price
+    createPublicClient: (...args: [any]) => {
+      const client = _viem.createPublicClient(...args)
+      client.getGasPrice = async () => parseGwei('0.00000042')
+      return client
+    },
+  }
+})
+
+// using this optimist https://optimistic.etherscan.io/tx/0xaa291efba7ea40b0742e5ff84a1e7831a2eb6c2fc35001fa03ba80fd3b609dc9
+const blockNumber = BigInt(107028270)
```

Comment:
```suggestion
const blockNumber = 107028270n
```

Reply:
Keeping open because if I merge it into github remote I might accidentally force push the change away but will do